common/serialization/subtitles/subtitles.cpp

```diff
@@ -253,6 +253,10 @@ void parse_subtitle(const goos::Object& data,
         auto kind = SubtitleSceneKind::Movie;
         int id = 0;
         auto entries = cdr(obj);
+        if (!(entries.is_pair())) {
+          lg::warn("{} | Not a pair! Empty movie subtitle?", head.as_string()->data);
+          return;
+        }
         if (head.is_int()) {
           kind = SubtitleSceneKind::Hint;
         } else if (car(entries).is_symbol()) {
```

Comment:
I took a closer look and I see the issues you were talking about now. You can get rid of this entire `!entries.is_pair()` block earlier and simply add it to this line here so that it's `entries.is_pair() && car(entries).is_symbol()`. This was my bad I think.